### PR TITLE
BRAYNS 520 - Support to export G-Buffers

### PR DESCRIPTION
**Pull request by NadirRoGue**

### CODE REVIEW

File: brayns/engine/framebuffer/types/StaticFrameHandler.cpp [hunk 1]
```diff
@@ -23,35 +23,96 @@
...
  * @brief Returns the size, in bytes, of each color channel of a given pixel format
  */
 class PixelFormatChannelByteSize
 {
 public:
-    static size_t get(brayns::PixelFormat format)
+    static size_t get(brayns::FramebufferChannel channel, brayns::PixelFormat format)
     {
-        switch (format)
+        if (channel == brayns::FramebufferChannel::Color)
```
Comment (Adrien4193) [inline brayns/engine/framebuffer/types/StaticFrameHandler.cpp:43]: Just maniac stuff but I would put if channel != Color return 4 and then the rest...

File: brayns/utils/image/ImageCodecRegistry.cpp [hunk 2]
```diff
@@ -80,19 +81,21 @@ class ImageCodecStorage
...
 } // namespace
 
 namespace brayns
 {
-const ImageCodec &ImageCodecRegistry::getCodec(std::string format)
+const ImageCodec &ImageCodecRegistry::getCodec(const std::string &format)
 {
-    StringCase::lower(format);
+    auto lowerFormat = format;
+    StringCase::lower(lowerFormat);
```
Comment (Adrien4193) [inline brayns/utils/image/ImageCodecRegistry.cpp:95]: Single line with StringCase::toLower(format) :)

**Review by Adrien4193 — DISMISSED** [review]

File: brayns/utils/image/ImageCodecRegistry.cpp (hunk 2 above)
Comment (NadirRoGue) [reply brayns/utils/image/ImageCodecRegistry.cpp]: Addressed

File: brayns/engine/framebuffer/types/StaticFrameHandler.cpp (hunk 1 above)
Comment (NadirRoGue) [reply brayns/engine/framebuffer/types/StaticFrameHandler.cpp]: Addressed

**Review by Adrien4193 — DISMISSED** [review]

**Review by Adrien4193 — APPROVED** [review]

**Pull request merged**